examples/convert_ieeg_to_bids.py

```diff
@@ -0,0 +1,163 @@
+"""
+===================================
+07. Convert iEEG data to BIDS format
+===================================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of iEEG
+data. Specifically, we will follow these steps:
+
+1. Download some iEEG data from the MNE-Python API
+   `PhysioBank database <https://physionet.org/physiobank/database>`_.
+    `https://mne.tools/stable/auto_tutorials/misc/plot_ecog.html#sphx-glr-auto-tutorials-misc-plot-ecog-py`_.
+
+2. Load the data, extract information, and save in a new BIDS directory
+
+3. Check the result and compare it with the standard
+
+The iEEG data will be pretty similar to the iEEG data with
+the addition of extra elements in the electrodes.tsv and
+coord_system.json files.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+# License: BSD (3-clause)
+
+import os
+import tempfile
+
+# need to install visualization
+from mpl_toolkits.mplot3d import Axes3D  # noqa
+import matplotlib.pyplot as plt
+import mne
+import numpy as np
+from scipy.io import loadmat
```

Comment:
order of imports is not correct. First std lib, then numpy/scipy/matplotlib and then mne and then mne_bids. From the more general to the specific